src/input.rs

```diff
@@ -1136,12 +1136,12 @@ impl<'a, 'parse, I: Input<'a>, E: ParserExtra<'a, I>> InputRef<'a, 'parse, I, E>
         unsafe { self.input.span(before.offset..self.offset) }
     }
 
-    #[cfg(feature = "regex")]
     #[inline(always)]
-    pub(crate) fn skip_bytes<C>(&mut self, skip: usize)
+    #[cfg(feature = "regex")]
+    #[cfg(feature = "lexical-numbers")]
```

Comment:
I think `cfg(any(feature = "regex", feature = "lexical-numbers"))` is the proper thing to do here.